src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -19,7 +25,24 @@ public ExceptionRecord mapEnvelope(Envelope envelope) {
             envelope.jurisdiction,
             getLocalDateTime(envelope.deliveryDate),
             getLocalDateTime(envelope.openingDate),
-            mapDocuments(envelope.documents)
+            mapDocuments(envelope.documents),
+            mapOcrData(envelope)
         );
     }
+
+    private List<CcdCollectionElement<CcdKeyValue>> mapOcrData(Envelope envelope) {
```

Comment:
Can we just pass `ocrData` here instead? function doesn't really need anything from envelope - just map ocr data to something else

Reply:
We can, will do